versions.props

```diff
@@ -6,7 +6,7 @@
         <RxVersion>4.4.1</RxVersion>
         <ProtobufVersion>3.18.1</ProtobufVersion>
         <GrpcVersion>2.41.0</GrpcVersion>
-        <MongoDBDriverVersion>2.10.1</MongoDBDriverVersion>
+        <MongoDBDriverVersion>2.13.2</MongoDBDriverVersion>
```

Comment:
Just need to check to be sure - this is compatible with the MongoDB we provide in the platform right?

Reply:
It is :)